core/src/main/kotlin/org/evomaster/core/Main.kt

```diff
@@ -339,9 +339,16 @@ class Main {
             assert(controllerInfoDto==null || controllerInfoDto.fullName != null)
             val solutions = TestSuiteSplitter.split(solution, config.testSuiteSplitType)
             writer.setSwagger(swagger)
-            solutions.forEach {
+            /*solutions.forEach {
                 if(it.individuals.isNotEmpty()) writer.writeTests(it, controllerInfoDto?.fullName)
             }
+
+             */
+
+            solutions.filter { !it.individuals.isNullOrEmpty() }.map {
+                writer.writeTests(it, controllerInfoDto?.fullName)
```

Comment:
that's weird. why a `map` instead of a `forEach`?
furthermore, for readability, the `.forEach{` would be best on next line instead of being 
on same line as `.filter{`

Reply:
Just pushed a fix for it. It was mostly a leftover from a previous version.